### ISSUE TITLE
googleway:Google_places - problems with page_token

### ISSUE THREAD

**Issue by pmasotti** [SymbolixAU/googleway#36]
Hi. I'm using google_places of   googleway package.
I am triying to loop to the next pages using the page_token;
I put as input the df_temp$next_page_token.
The first two loops work. 
The third return dataframe$next_page_token = NULL
Is it a problem with the interface?
Thanks

**Comment by SymbolixAU** [comment]
Can you share your code that's causing the issue?

**Comment by pmasotti** [comment]
I have worked on the issue ad finally get the response from google api @
https://developers.google.com/places/web-service/search?hl=it
"next_page_token contains a token that can be used to return up to 20
additional results.  A next_page_token will not be returned if there are no
additional results to display. The maximum number of results that can be
returned is 60. There is a short delay between when a next_page_token is
issued, and when it will become valid."
For what I have tested, the package works correctly, and after the third
call to the API, produces the next_page_token = NULL according to
GoogleAPI/Place_search/Documentation.



2017-03-16 4:02 GMT+01:00 SymbolixAU <notifications@github.com>:
